android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/components/FormAnswer.kt

```diff
@@ -11,36 +11,62 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import co.nimblehq.ic.kmm.suv.android.ui.theme.AppTheme
 import co.nimblehq.ic.kmm.suv.android.ui.theme.Typography
+import co.nimblehq.ic.kmm.suv.domain.model.Answer
+import co.nimblehq.ic.kmm.suv.domain.model.AnswerInput
+import co.nimblehq.ic.kmm.suv.domain.model.Answerable
+import co.nimblehq.ic.kmm.suv.domain.model.getContentType
 
-typealias Placeholder = String
+object FormAnswerContentDescription {
+    private const val TEXT_FIELD = "TEXT_FIELD"
+
+    fun field(index: Int) = "$TEXT_FIELD-$index"
+}
 
 @Composable
 fun FormAnswer(
-    placeholders: List<Placeholder>,
-    onTextChange: (Pair<Int, String>) -> Unit,
-    modifier: Modifier = Modifier
+    answers: List<Answerable>,
+    onInputChange: (List<AnswerInput>) -> Unit,
+    modifier: Modifier = Modifier,
+    input: List<AnswerInput> = emptyList()
 ) {
+    val defaultInputs: List<AnswerInput> = answers.map { AnswerInput.Content(it.id, "") }
+    var changes by remember {
+        if (input.isEmpty()) {
+            mutableStateOf(defaultInputs)
+        } else {
+            mutableStateOf(input)
+        }
+    }
```

Comment:
```suggestion
    var changes by remember { mutableStateOf(input.ifEmpty { defaultInputs }) }
```

Reply:
Amazing suggestion! Fixed in https://github.com/suho/kmm-ic/pull/123/commits/07f6a3df329b5ed79084a3fe821669529cd47d33